### ISSUE TITLE
Prey-Predator interaction

### ISSUE THREAD

**Issue by ntromas** [zdk123/SpiecEasi#178]
Hi, 

Do you think that SpiecEasi could reveal potential predator–prey interactions (cross-domain)? If not, do you have in mind an approach to specifically analyzed these interactions in a microbial community (still using a cross domain approach, so basically two datasets).

Thanks !

Nico

**Comment by zdk123** [comment]
are both datasets compositional?

**Comment by ntromas** [comment]
Yes. With the same DNA, we used two different markers to generate these datasets.

**Comment by zdk123** [comment]
Yep you can try [cross-domain](https://github.com/zdk123/SpiecEasi#cross-domain-interactions) mode

**Comment by ntromas** [comment]
We already tried but not sure how to reveal the potential interactions that
would fit with a LV model. I wonder how spieceasi would deal with this kind
of interactions.

The current LV analysis (Mdcine,MetaMis,...) were not designed to deal with
2 compositions.

Thanks for the suggestions/advices,



Le lun. 9 août 2021 22 h 48, Zachary Kurtz ***@***.***> a
écrit :
